src/Components/DatePicker/Component.purs

```diff
@@ -105,16 +105,19 @@ component =
     { initialState
     , render
     , eval
-    , receiver: HE.input Receive
+    , receiver: const Nothing
     , initializer: Just $ H.action Initialize
     , finalizer: Nothing
     }
   where
     initialState :: Input -> State
     initialState { targetDate, selection } =
-      { targetDate: fromMaybe (Tuple (ODT.unsafeMkYear 2001) (ODT.unsafeMkMonth 1)) targetDate
+      let targetDate' = fromMaybe (Tuple (ODT.unsafeMkYear 2001) (ODT.unsafeMkMonth 1)) targetDate
+        in
```

Comment:
Personally I prefer `do, let` syntax to `let ... in`, I think it's cleaner and easier to format

Reply:
So the above would instead be
```purescript
    initialState { targetDate, selection } = do
      let targetDate' = fromMaybe (Tuple (ODT.unsafeMkYear 2001) (ODT.unsafeMkMonth 1)) targetDate
```